<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>873</width>
    <height>233</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_Path">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>741</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Path">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>10</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>경로열기</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_FileList">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>60</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>파일리스트</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_FileList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>741</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
